src/magic_folder/scripts/magic_folder_cli.py

```diff
@@ -3,8 +3,10 @@
 import os
 import sys
 import urllib
+from six.moves import (
+    StringIO as MixedIO,
+)
```

Comment:
Should not be a behavior change but should be more clear to readers.  This is not a byte stream.  This is not a text stream.  This is a whatever-you-feel-like stream.  This is bad and should be fixed but the implementation makes no consistent decision about whether it wants to write out bytes or text  and I didn't want to fix every string literal in the whole project in this PR.